lib/npm.js

```diff
@@ -167,10 +168,10 @@ const npm = module.exports = new class extends EventEmitter {
 
   async [_load] () {
     process.emit('time', 'npm:load:whichnode')
-    const node = await which(process.argv[0]).catch(er => null)
+    const node = which.sync(process.argv[0])
```

Comment:
do we want this to throw if it errors? previously we were ignoring failure, but this will start bubbling them out instead. the switch from async to sync here is maybe notable, but also not really functionally relevant i don't think

Reply:
Oooh good catch.  No pun intended.